pcsx2/SIO/Memcard/MemoryCardFile.cpp

```diff
@@ -309,17 +306,12 @@ void FileMemoryCard::Open()
 
 		if (!m_file[slot])
 		{
-			// Translation note: detailed description should mention that the memory card will be disabled
-			// for the duration of this session.
-			Host::ReportFormattedErrorAsync("Memory Card", "Access denied to memory card: \n\n%s\n\n"
-														   "Another instance of PCSX2 may be using this memory card. Close any other instances of PCSX2, or restart your computer.%s",
-				fname.c_str(),
-#ifdef WIN32
-				"\n\nIf your memory card is in a write-protected folder such as \"Program Files\" or \"Program Files (x86)\", move it to another folder, such as \"Documents\" or \"Desktop\"."
-#else
-				""
-#endif
-			);
+			Host::ReportErrorAsync(TRANSLATE_SV("MemoryCard", "Memory Card Read Failed"),
+				fmt::format(TRANSLATE_FS("MemoryCard", "Unable to access memory card: \n{}\n\n"
+													   "Another instance of PCSX2 may be using this memory card "
+													   "or the memory card is stored in a write-protected folder.\n"
+													   "Close any other instances of PCSX2, or restart your computer\n."),
```

Comment:
```suggestion
													   "Close any other instances of PCSX2, or restart your computer.\n"),
```
period before \n, not after